src/figuresClasses.ts

```diff
@@ -1,19 +1,72 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea: () => number;
 }
 
 export class Triangle implements Figure {
+  shape = 'triangle';
+
+  color: string;
+
+  constructor(
+    color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0 || a + b <= c || a + c <= b || b + c <= a) {
+      throw new Error('Invalid triangle sides');
+    }
+    this.color = color;
```

Comment:
```suggestion
  constructor(
    public color: string,
    public a: number,
    public b: number,
    public c: number,
  ) {
    if (a <= 0 || b <= 0 || c <= 0 || a + b <= c || a + c <= b || b + c <= a) {
      throw new Error('Invalid triangle sides');
    }
```
looks like you can simplify it